os_update-formula/pillar.example

```diff
@@ -0,0 +1,18 @@
+os-update:
+  # by default, the timer is left disabled
+  # to enable it, set this to true
+  enable: false
```

Comment:
The default in map.jinja disagrees with what the comment says ;-)

Enabling the timer by default is a good idea IMHO (why else would you include this formula?), so please update pillar.example.

Reply:
Wonder how that happened - thanks, corrected.